docs/adrs/0080-use-flipt-feature-flags.md

```diff
@@ -0,0 +1,106 @@
+
+Scenario 1: The Bug vs. The Incomplete Feature
+
+While one of our teams is part-way through an epic that adds a new feature, a critical bug is reported. A fix for the bug is quickly found and committed. But if we deploy code from the main branch at that point, half of the new feature will also be included, since it was already committed. Releasing half of a feature to production is generally frowned upon.
+
+Scenario 2: Changing Authentication
+
+After we have been live for several months, the customer requires us to move from using login.gov to authenticate users to Okta. Writing and testing the code is no problem. But when we deploy it, every user now has to set up a new authentication account. Many of those users will be in the middle of moving to their next duty station when we spring this on them. Users are constantly moving, there will not be a break in the schedule where we can do the changeover.
+
+Scenario 3: Prime Freeze
+
+For 30 days before we go live, the Prime (HSA) requires a code freeze for testing. We do not want to stop deploying for 30 days, and we really don't want to stop development for 30 days. Feature flags would allow us to build (and maybe even deploy) code during the freeze and remain productive.
+
+## Considered Alternatives
+
+* Option #0: Do Nothing. Stay the course, don't make any changes, figure out what to do when problems arise
+* Option #1: LaunchDarkly: This is the industry standard feature flag platform.
+* Option #2: Unleash: Open source, can be self-hosted
+* Option #3: AWS AppConfig: we're already using lots of AWS, what's a little more?
+* Option #4: Roll our own
+* Option #5: Flipt: Open source, self hosted, supports file based config
+
+## Decision Outcome
+
+Chosen Alternative: Option #5: Flipt
+
+LaunchDarkly is the industry standard, but there is no room for it in the budget. Unleash has similar features to AWS AppConfig, but requires more setup and maintenance effort when self-hosted (e.g. it requires a PostgreSQL database). AWS AppConfig is really a thin shim over config files and doesn't provide much in the way of helping us manage feature groupings.
+
+Rolling our own is a possibility, but one key feature we'd want to think about is how to promote feature flags from one environment to another (e.g. from staging to production). That suggests that tying the feature flag to the environment by e.g. storing the flags in the database isn't ideal as we need to recreate the flag settings in each environment. Combining that with the desire to have feature flags be enabled on a per user basis, it makes the complexity of rolling our own outweigh the relatively simple flipt deployment.
```

Comment:
**suggestion:** I'd want to put these two paragraphs into their own respective section or with the option since I feel like it's confusing to read 
Chosen alternative: Option #5: Flip 
<content that is not about Flip>

Reply:
Huh, that's interesting. I was trying to use this section to give an overview of why Flipt was chosen, and so that means it touches on all of the options, not just flipt.

Reply:
Ohhh I see, using this section in this manner makes sense. I think it was a bit confusing for me when I read the heading / subheading and then the content, but if other folks are ok with this, then lets leave the section organized as is.

Reply:
I've added a sentence at the beginning of this section to indicate it is a summary of the decision. Maybe that's better?